app/Unit.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace ConverterApp;
+
+class Unit
+{
+    protected $longName;
+    protected $shortName;
+
+    public function getLongName()
+    {
+        return $this->longName;
+    }
+    public function setLongName($lName)
```

Comment:
Using full name is recommended. We should avoid anything which could lead to confusion. `lName` should be `longName`